platforms/pf_test/platform.cpp

```diff
@@ -284,6 +294,23 @@ void pf_init_tasks(void)
     bool start_planner = true;
     int countdown = PF_START_COUNTDOWN;
 
+    /* Initialize UARTPB */
+    uartpb = new cogip::uartpb::UartProtobuf(
+        message_handler,
+        UART_DEV(1)
+        );
+
+    bool uartpb_res = uartpb->connect();
+    if (! uartpb_res) {
+        uartpb = nullptr;
+        cogip::tracefd::out.logf("UART initialization status: %d\n", uartpb_res);
```

Comment:
"UART initialization failed, error: %d\n"

Reply:
Done, for cortex platform too.
Commit message for cortex platform was wrong too, also fixed (cup2021 => cortex).